lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -70,11 +79,11 @@ foreach ($member in $members) {
     }
 
     if ($state -in @("present", "pure") -and !$user_in_group) {
-        Add-ADGroupMember -Identity $name -Members $group_member -WhatIf:$check_mode @extra_args
+        Add-ADPrincipalGroupMembership -Identity $group_member -MemberOf $name -WhatIf:$check_mode @extra_member_args
```

Comment:
I take it the change from `ADGroupMember` to `ADPrincipalGroupMembership` was done because we cannot add another domain's member using the former whereas the latter is?

Reply:
This is correct i was not able to do it with ADGroupMember and wenever i search for this particular issue in multidomain forrest i always find ADPrincipalGroupMembership.